src/Components/ProjectTile.js

```diff
@@ -0,0 +1,103 @@
+    updatedProduct.downVoted = false;
+    let newVote = updatedProduct.upVoted ? 1 : (updatedProduct.downVoted ? -1 : 0);
+    updatedProduct.votes = updatedProduct.votes - currentVote + newVote;
+    setProducts(products.map((product) => product.id === products[index].id ? updatedProduct : product));
+  };
+  const downVote = () => {
+    const updatedProduct = { ...products[index] };
+    let currentVote = updatedProduct.upVoted ? 1 : (updatedProduct.downVoted ? -1 : 0);
+    updatedProduct.downVoted = !updatedProduct.downVoted;
+    updatedProduct.upVoted = false;
+    let newVote = updatedProduct.upVoted ? 1 : (updatedProduct.downVoted ? -1 : 0);
+    updatedProduct.votes = updatedProduct.votes - currentVote + newVote;
+    setProducts(products.map((product) => product.id === products[index].id ? updatedProduct : product));
+  };
+  const goTo = (product) => () => {
+    history.push(`/${product}/getFeature`);
+  };
+  const capitalizeFirstLetter = (string) => {
+    return string.charAt(0).toUpperCase() + string.slice(1);
+  };
+
+  const deleteDiv = (event) => {
+    const product = event.target.parentNode.parentNode.parentNode;
+    product.parentNode.removeChild(product);
+
+  };
+  return (
+    <div className="child product">
+      <div className="product-container">
+        <div className="image-container">
+          <img src={products[index].image_url} alt={products[index].name} />
+        </div>
+        <div className="content">
+          <div className="product-content">
+            <span className="product-title" 
+            data-testid={"ptnav:"+index}
+            onClick={goTo(products[index].name)} 
+            style={{ marginTop: 'auto', marginBottom: 'auto' }}>
+              {capitalizeFirstLetter(products[index].name)}
+            </span>
+            <p className="product-description">
+              {products[index].description}
+            </p>
+          </div>
+          <div className="tag-container">
+            {products[index]['tags'].map(tag =>
+              <div key={tag}>
+                <span className="tag">{tag.toUpperCase()}</span>
+                <div>&nbsp;</div>
+              </div>
+            )}
+          </div>
+        </div>
+        <div className="votes-container">
+          <span>
+            <FontAwesomeIcon icon={faChevronUp} size="lg" 
+            className={products[index].upVoted ? 'votedUp' : 'voteup'} 
+            data-testid={"pt_up:"+index}
+            onClick={upVote} />
+          </span>
+          <span>
+            {products[index].votes}
+          </span>
+          <span>
+            <FontAwesomeIcon icon={faChevronDown} 
+            size="lg" 
+            className={products[index].downVoted ? 'votedDown' : 'votedown'} 
+            data-testid={"pt_down:"+index}
+            onClick={downVote} />
+          </span>
+        </div>
+        <br/>
+        <div id ="delete_button" className="delete_project" style={{marginLeft:'25px'}}>
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/elric97/feature-hunt/pullRequest?prid=8494981)
```suggestion
        <div id ="delete_button" className="delete_project" style={{marginLeft:"25px"}}>
```